model/feature_flags.go

```diff
@@ -109,6 +111,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.AnnualSubscription = false
 	f.ReduceOnBoardingTaskList = false
 	f.ThreadsEverywhere = false
+	f.SelfHostedFirstTimePurchase = false
```

Comment:
Given that for self hosted we can't control the feature flags with Split.io, we may need to default this to `true` to be able to have the feature released - unless there is plans to add to a configuration which I'm not sure if what was intended here